### PR TITLE
Normalize label keys

### PR DESCRIPTION
**Pull request by shouichi**
Close https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/98.

### CODE REVIEW

File: exporter/metric/metric.go [hunk 1]
```diff
@@ -604,3 +604,15 @@ func aggToTypedValue(kind apimetric.NumberKind, value apimetric.Number) (*monito
...
+// https://github.com/googleapis/googleapis/blob/c4c562f89acce603fb189679836712d08c7f8584/google/api/metric.proto#L149
+//
+// > The label key name must follow:
+// >
+// > * Only upper and lower-case letters, digits and underscores (_) are
+// >   allowed.
+// > * Label name must start with a letter or digit.
+// > * The maximum length of a label name is 100 characters.
+func normalizeLabelKey(s string) string {
+	return strings.ReplaceAll(s, ".", "_")
```
Comment (nilebox) [inline exporter/metric/metric.go:617]: This looks good as the first step, though Ideally we would also need to make sure that the label key is valid after this conversion (with all the other limitations you listed above), and might be useful to return an error if it's still invalid after "normalization".

**Review by nilebox — APPROVED** [review]

File: exporter/metric/metric.go (hunk 1 above)
Comment (shouichi) [reply exporter/metric/metric.go:617]: Yes. We try to adopt this exporter to our production and we might send another PR(s) in the future. Thanks for the quick merge 👍

**Comment by shouichi** [comment]
Tried 45880ec6f71fcaf0b3830da8361be24a715646d8 and confirmed that writing timeseries worked 🎉

File: exporter/metric/metric.go (hunk 1 above)
Comment (martinraison) [reply exporter/metric/metric.go:617]: This is still relevant today. Especially with third-party library instrumentation, it's not always trivial to guarantee all metric names follow Google's conventions. For now I'm replacing all characters outside `[A-Za-z0-9]` with `_` and truncating to 100 characters

**Pull request merged**